src/core/dbus-execute.c

```diff
@@ -2366,7 +2358,7 @@ int bus_exec_context_set_transient_property(
                                                 u, flags|UNIT_ESCAPE_SPECIFIERS, name,
                                                 "%s=%s%s:%s:%s",
                                                 name,
-                                                ignore_enoent ? "-" : "",
+                                                !!ignore_enoent ? "-" : "",
```

Comment:
not that it matters, but the `!!` here is redundant... `!!` is usually used to turn a non-boolean into a boolean value, i.e. 0 or 1. But the ternary operator does that anyway for the test...